### DISCUSSION TITLE
How to set strings for failed CI checks of Scheduled Reminders

### DISCUSSION THREAD

**Discussion by kenta-polyglot** [community/community#23986] [New to GitHub]
<p>Hi,</p>
<p>In <a href="https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/managing-your-scheduled-reminders#about-scheduled-reminders-for-users" rel="nofollow noopener">Scheduled Reminder for user</a> settings, I want to be notified when CI of Pull Request fails</p>
<p>However, there are very few setting examples of “Your PR has failed CI checks”, so I can’t understand how to do it if I want to set the workflow of GitHub Actions here.</p>
<p>My GitHub Actions workflow is shown below on the Pull Request screen.<br>
<img src="https://user-images.githubusercontent.com/1457528/181080513-88792ed1-834f-4bc1-bd96-7700a980a3ff.png" alt="スクリーンショット 2020-07-14 12.55.47" data-base62-sha1="z7LojzrhIXK0E7uFNDyYsjCEvFv" width="683" height="280"></p>
<p>I would be grateful if anyone could tell me the correct setting.</p>

**Comment by Jaboo9** [comment, marked as answer]
<p>Hi</p>
<p>The issue with receiving slack notifications for failed CI checks has been fixed. You can check again to confirm.</p>

**Reply by sasacocic** [reply]
I don't think this should be the answer. It doesn't show any examples of how to actually set the string.

**Reply by Gruebes** [reply]
Hope this helps: https://github.com/orgs/community/discussions/23986#discussioncomment-6826037

**Reply by mateusloubach** [reply]
Most of these accepted answers have been this. some text or a link and voilá.. accepted answers, while some dude spent the past half hour helping step-by-step and gets no upvote at all..

**Comment by kenta-polyglot** [comment]
<p><a class="mention" href="/jaboo9">@Jaboo9</a></p>
<p>Thanks a lot! I will check again!</p>

**Comment by eggilbert** [comment]
<p>Any luck with this? I’m running into this problem too, and just tried again and couldn’t get any permutation of CI check name to work.</p>
<p>It’s unclear what the CI check name is expected to be. What I see in the UI on the pull request is prefixed with the GitHub App name, but referencing one in the protected branch settings on the repo it’s only the latter suffix that shows up in the list.</p>
<p>I’m not using GitHub Actions, but I tried the equivalent of:</p>
<ul>
<li><code>check</code></li>
<li><code>Check / check</code></li>
<li><code>Check / check (pull_request)</code></li>
<li><code>"check"</code></li>
<li><code>"Check / check"</code></li>
<li><code>"Check / check (pull_request)"</code></li>
<li><code>check, Check / check, Check / check (pull_request)</code></li>
</ul>
<p>And in all of these cases when the build fails and the status is updated, there’s no notification in Slack. Other real-time alerts are working, it’s just the CI check one that isn’t triggering.</p>

**Comment by kenta-polyglot** [comment]
<p><a class="mention" href="/eggilbert">@eggilbert</a></p>
<p>Hi,  My case is fixed and Slack is now notified when CI fails.</p>
<p>My CI name on pull request is “Check / check (pull_request)” and I  have entered “check” for “Your PR has failed CI checks” in the scheduled reminder.</p>
<p>I hope it will be helpful.</p>

**Comment by gnumoksha** [comment]
<p>My checks are named like in:</p>
<ul>
<li>ci/circleci: build-project</li>
<li>ci/circleci: libs-unit-tests</li>
</ul>
<p>I had to put all of them, separated by comma, like in <code>ci/circleci: build-project,ci/circleci: libs-unit-tests</code></p>
<p>I must say this feature worked great before it was integrated into GitHub.</p>

**Comment by maximevaillancourt** [comment]
I can't use the "Your PR has failed CI checks" real-time alerting because the status checks I'm trying to use it with have commas in their names (e.g. `Test / Tests (8, 0)`). Those status check names are generated by GitHub's own "matrix" feature in GitHub Actions, and it doesn't seem possible to rename those.

In other words, one of GitHub's features (GitHub Actions runner matrix) prevents using another GitHub feature (the "Your PR has failed CI checks" real-time alerting) because of the commas in the auto-generated status check names.

Is there a way to circumvent this issue somehow? I tried escaping the commas, wrapping the name in double quotes, and combining the two, to no avail:

<img width="477" alt="image" src="https://github.com/community/community/assets/8457808/f6c1da2c-894f-4dd6-b1fd-9639f3ce3fe4">

----

**Potential next steps for GitHub's product team**
[The product feedback in this comment is excellent](https://github.com/orgs/community/discussions/12511#discussioncomment-3791050) and I would love to GitHub address it!

**Comment by Gruebes** [comment]
I was able to get this to work using the job name from my yaml file:

![image](https://github.com/community/community/assets/9757631/9d9ff432-ce25-4270-91fc-2905c8de7dfa)

Using the value `build` ( NOT `Build and Test` from the property `name:`) - Example below

<img width="504" alt="image" src="https://github.com/community/community/assets/9757631/ebecd313-989f-4d6c-ae03-5ebb6e019385">

I received a notification when the pr build failed 🎉

![image](https://github.com/community/community/assets/9757631/9245d8b1-99ef-401f-9ab7-afd180711743)

**Comment by Joel-RS21** [comment]
How do I do "all"?!

**Reply by irl-segfault** [reply]
honestly hilarious that there isn't an easy way to do this

**Reply by yasht01** [reply]
much needed feature.

**Reply by syed-dawood** [reply]
> How do I do "all"?!

Did anybody find a solution?
